src/main/scss/components/_status-icons.scss

```diff
@@ -0,0 +1,43 @@
+.status-icon {
+  --status-background-opacity: 0.125;
+  --status-border-opacity: 0.25;
```

Comment:
I was also working on something similar to this PR (https://github.com/jenkinsci/jenkins/compare/master...janfaracik:jenkins:refine-build-status-icons?expand=1)

Would be good to get the below in so that the icons' contrast increases when necessary.

```
@media (prefers-contrast: more) {
    --status-background-opacity: 0;
    --status-border-opacity: 1;
  }
```